plugins/playwright/index.js

```diff
@@ -0,0 +1,29 @@
+
+const fs = require("fs");
+const path = require("path");
+
+// load the library in UMD format which self executes and adds window.querySelectorShadowDom
+const querySelectorShadowDomUMD = fs.readFileSync(path.resolve(__dirname, "../../dist/querySelectorShadowDom.js"))
+
+// a string because playwright does a .toString on a selector engine and we need to
+// make sure that query-selector-shadow-dom is injected and loaded into the function closure
+const engineString =`
+    options = options || {};
+    const name = options.name;
```

Comment:
Please add a default name of the selector engine.
This way we can register the selector engine without having to set an optional name parameter.